lib/grandpa/grandpa.go

```diff
@@ -550,6 +553,8 @@ func (s *Service) sendVoteMessage(stage Subround, msg *VoteMessage, roundComplet
 	ticker := time.NewTicker(s.interval * 4)
 	defer ticker.Stop()
 
+	// Eventhough, this looks like we are sending messages multiple times,
```

Comment:
nit not sure what you mean by `Eventhough`?

Reply:
```suggestion
	// Though, this looks like we are sending messages multiple times,
```

Reply:
call me Grammarly :smile: , but starting a sentence with `Though` doesn't make much sense, use `Although` instead :wink: (and also remove that comma)